.editorconfig

```diff
@@ -0,0 +1,22 @@
+# top-most EditorConfig file
+root = true
+
+# Apply to all files
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+# Kotlin specific rules
+[*.{kt,kts}]
+max_line_length=120
+
+# Ruby specific rules
+[*.rb]
+indent_style = space
+indent_size = 2
+
+# Fastlane specific rules
+[**Fastfile]
+indent_style = space
```

Comment:
I'm wondering if we could not group the `Fastfile` in the same group as `*.rb` (after all, Fastfiles are ruby files) to ensure the same rules in both without having to repeat everything.

Also, I think we should applying the same rules to `Gemfile` while we're at it.

Reply:
I remember trying this at the time, but iirc it wasn't working in my vim setup. I might just need to tweak something in my setup, but thought I'd mention it in case @loremattei hits a similar issue.

Reply:
Good point @AliSoftware!
I've updated the configuration in [d088c4b](https://github.com/wordpress-mobile/release-toolkit/pull/219/commits/d088c4be98928e05e9978b94e28e15dcd060235f).

@oguzkocer this seems to work for me on VSCode and MacVim. Please let me know if it doesn't work on your configuration. It breaks the purpose of this file if it doesn't work on every environment 😅, so I'll revert to the previous configuration in that case.